token_store.go

```diff
@@ -567,6 +585,41 @@ func (ts *TokenStore) GetBasicIDByAccess(ctx context.Context, access string) (st
 	return basicID, nil
 }
 
+// GetOriginalBasicIDByAccess returns the original token ID of basic token info based on access token
+func (ts *TokenStore) GetOriginalBasicIDByAccess(ctx context.Context, access string) (originalID string, err error) {
```

Comment:
is it worth it to have this function when you can call the one below and ignore the first variable?